src/main/java/com/ericsson/ei/config/SwaggerConfig.java

```diff
@@ -33,25 +34,30 @@
 @EnableSwagger2
 public class SwaggerConfig {
 
+    private static final String CONTACT_NAME = "Eiffel Intelligence Maintainers";
+    private static final String CONTACT_URL = "https://github.com/eiffel-community/eiffel-intelligence";
+    private static final String CONTACT_EMAIL = "https://groups.google.com/forum/#!forum/eiffel-community";
```

Comment:
can we add eiffel-community@googlegroups.com

Reply:
Sure, I'll change it